index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div>
+        <form id="new-task-form" action="/submit" method="post">
+          <label for="name">Name:</label>
+          <input type="text" id="name" name="new-task-input" required />
```

Comment:
Let's make the input box a textarea.
```suggestion
          <textarea type="text" id="todo-input-box" name="new-task-input" required />
```